cves/CVE-2011-0722.yml

```diff
@@ -203,8 +239,10 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    No specification violations are mentioned in the CVE, bug report, relevant
+    commit messages, or security advisories for package updates.
+  answer: false
```

Comment:
Double-checked this, seems good!

Reply:
Cool - glad I probably didn't miss anything.